save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/ProjectController.kt

```diff
@@ -56,16 +57,21 @@ class ProjectController(
     private val projectPermissionEvaluator: ProjectPermissionEvaluator,
     private val lnkUserProjectService: LnkUserProjectService,
 ) {
-    @GetMapping("/all")
+    @PostMapping("/all-by-filters")
     @RequiresAuthorizationSourceHeader
     @PreAuthorize("hasRole('ROLE_SUPER_ADMIN')")
     @Operation(
-        method = "GET",
+        method = "POST",
         summary = "Get all projects.",
         description = "Get all projects, including deleted and private. Only accessible for super admins",
     )
+    @Parameters(
+        Parameter(name = "projectFilters", `in` = ParameterIn.DEFAULT, description = "project filters", required = true),
+    )
     @ApiResponse(responseCode = "200", description = "Projects successfully fetched.")
-    fun getProjects(): Flux<Project> = projectService.getProjects()
+    fun getProjects(
+        @RequestBody(required = true) projectFilters: ProjectFilters
```

Comment:
`(required = true)` is value by default, no need to specify it explicitly

Reply:
Deleted